tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "root": true,
   "extends": "@fullstacksjs/tsconfig/react",
-  "include": ["src", "@types", "vite.config.ts"],
+  "include": ["src", "@types", "./**/*"],
```

Comment:
I don't recommend doing `./**/*`, because some plugins which have `.config.ts` file are not compatible with our config (`"module": "ESNext"`)